scripts/add-at-macro/README.md

```diff
@@ -32,43 +32,53 @@ times on a codebase.
 
 1. Install [babashka](https://github.com/babashka/babashka) `0.3.2` or later by following [these instructions](https://github.com/babashka/babashka#installation).
  
-1. Clone re-com's GitHub repository,
+2. Clone re-com's GitHub repository,
 
    ```
    git clone https://github.com/day8/re-com.git 
    ```
   
-2. Navigate to the scripts location
+3. Navigate to the scripts location
    ```
-   cd re-com/scripts/add-at-macro/src/add_at_macro
+   cd re-com/scripts/add-at-macro
    ```
 
-3. Run
+4. Run
 
    If the project using re-com had sources in `../my-project/src`, then run via babashka (aka `bb`):
    ```
-   bb core.clj "../my-project/src" 
+   bb add_at_macro.clj "../my-project/src" 
+   ```
+   
+   ##### Tip
+   
+   For run, (not test command in the next section) The `bb` command also takes the following extra command line 
+   arguments after the directory. (Order matters)
+   1. `-verbose`. When this is passed, the changes the script makes are printed to console. Example command
```

Comment:
Please add docopt so order doesn't matter, and we get command line help etc.

You can add this with babashka.deps prior to your ns form. E.g.

```clojure
(require '[babashka.deps :as deps])

(deps/add-deps
  '{:deps {docopt/docopt   {:git/url "https://github.com/nubank/docopt.clj"
                            :sha     "12b997548381b607ddb246e4f4c54c01906e70aa"}}})
                            
 (ns add-at-macro
    [docopt.core        :as docopt] )

;; see http://docopt.org/ for syntax     
def usage "Add at Macro Script

Usage:
  add_add_macro <directory> [options]

Options:
  -v --verbose Changes this script makes are printed to the console.
  -t --testing Changes this script makes are not written to disk, only printed to the console.
  -h --help     Show this screen.
  --version     Show version.
  ")

(docopt/docopt usage
        *command-line-args*
        (fn [arg-map]
          (clojure.pprint/pprint arg-map))))                               
```